.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6
+    - name: Install dependencies
+      run: |
+        gem install bundler
+        bundle install
+    - name: Run tests
+      run: |
+        bundle exec ruby test_data.rb
+
+  stations-human-diff:
+    name: Stations Human Diff
+    runs-on: ubuntu-latest
+    steps:
+    - uses: ftuyama/stations-human-diff@master
```

Comment:
trainline-eu here no?

Reply:
I need to move my fork to `trainline-eu` organization